support-frontend/assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -99,4 +100,33 @@ export const tests: Tests = {
     targetPage: '/(uk|us|eu|au|ca|nz|int)/subscribe/digital$',
     optimizeId: 'emQ5nZJCS5mZkhtwwqfx5Q',
   },
+  paymentSecurityDesignTest: {
+    type: 'OTHER',
+    variants: [
+      {
+        id: 'control',
+      },
+      {
+        id: 'V1_securetop',
+      },
+      {
+        id: 'V2_securemiddle',
+      },
+      {
+        id: 'V3_securebottom',
+      },
+      {
+        id: 'V4_grey',
+      },
+    ],
+    audiences: {
+      ALL: {
+        offset: 0,
+        size: 1,
+      },
+    },
+    isActive: true,
+    independent: true,
+    seed: 10,
```

Comment:
@jlieb10 @tomrf1 are these settings correct?

Reply:
yes 💯

Reply:
👍 yes
please can you also add the same `targetPage` value as on the `stripeElementsRecurring` test.
This is a new feature which prevents users being needlessly assigned to a test on other parts of the support website.

Reply:
👍